### PR TITLE
docs: Consul k8s - Add WAN Federation guide to Vault section

### PR DESCRIPTION
**Pull request by jmurret**

### CODE REVIEW

**Comment by david-yu** [comment]
@jmurret I added some formatting suggestions, and pushed straight to the PR since there were a lot of them.

**Comment by jmurret** [comment]
@david-yu thank you.  I am just working out some kinks with the commands, but should be done soon.

File: website/content/docs/k8s/installation/vault/wan-federation.mdx [hunk 1]
```diff
@@ -0,0 +1,672 @@
...
+</CodeBlockConfig>
+
+
+####  dc2 Datacenter
+##### Install Vault Injector in your Consul k8s cluster
+A minimal valid installation of Vault Kubernetes must include the Agent Injector which is utilized for accessing secrets from Vault. 
+In dc2, we will have to configure `externalvaultaddr` value point to the external address of the Vault server in dc1.
+
+Change your Kubernetes current context to target dc2:
+```shell
```
Comment (david-yu) [inline website/content/docs/k8s/installation/vault/wan-federation.mdx:174]:
```suggestion
```shell-session
```

**Comment by karl-cardenas-coding** [comment]
@jmurret I don't feel this belong in the docs. This is a tutorial so it should live in https://github.com/hashicorp/tutorials . I'm happy to help you roll out there.

**Comment by trujillo-adam** [comment]
+1 that it is currently framed as as a tutorial rather than as reference documentation. 

It also requires a significant amount of editing so that it conforms our style.

I wonder, however, if some version of this can be added to the docs as an overview of the functionality. Or maybe we can incorporate the code snippets from this tutorial into the appropriate documentation topics as example configurations. 

Either way, we should move the content to a doc so we can put more thought into how and where to present the information.

**Pull request closed without merging**